tests/TestWinapp.py

```diff
@@ -429,6 +429,46 @@ def test_excludekey(self):
 
             # cleanup
             shutil.rmtree(dirname, True)
+            
+        # indices for tests when the ExcludeKey includes square brackets
```

Comment:
@livingentity 

The test_excludekey() function is long, and we want to avoid long functions. How about splitting these into their own test function like test_excludeke_square_bracket ? Another benefit is each function can run, even when the other function fails.

Reply:
Done.